app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(
+            self,
+            comfort_class,
+            clean_mark,
+            brand,
+    ):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+            self,
+            distance_from_city_center,
+            clean_power,
+            average_rating,
+            count_of_ratings,
+    ):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars):
+        cost = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                cost += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return cost
+
+    def calculate_washing_price(self, car):
+        diff_btw = (self.clean_power - car.clean_mark)
+        cost = car.comfort_class * diff_btw * self.average_rating
+        price = cost / self.distance_from_city_center
+        return round(price, 1)
+
+    def wash_single_car(self, car):
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rate):
+        rating = self.average_rating * self.count_of_ratings + rate
+        count = self.count_of_ratings + 1
```

Comment:
better write here self.count_of_ratings += 1 and use it not to perform double calculation